src/Config/ShieldOAuthConfig.php

```diff
@@ -42,7 +42,8 @@ class ShieldOAuthConfig extends BaseConfig
             'client_id'     => 'Get it from Google',
             'client_secret' => 'Get it from Google',
 
-            'allow_login' => true,
+            'allow_login'    => true,
+            'allow_register' => true,
```

Comment:
You just add this option for Google. What about other drivers(GitHub and Yahoo)?

Reply:
Currently, I am not using any other driver. so I can't test yet

Reply:
I don't think there is a problem. Please add for both (github and yahoo).

The last two things that need to be done:
1- Updating the documents in the section below: 
https://www.shield-oauth.codeigniter4.ir/add_other_oauth/
see https://github.com/datamweb/shield-oauth/blob/develop/docs/add_other_oauth.md

2- Updating file **NewShieldOauthGenerator** for the correct operation of command 
`php spark make:oauth`
see https://github.com/datamweb/shield-oauth/blob/develop/src/Commands/Generators/NewShieldOauthGenerator.php

Reply:
I've sent an update for this